compiler/rustc_middle/src/mir/interpret/error.rs

```diff
@@ -228,6 +228,10 @@ pub enum UndefinedBehaviorInfo<'tcx> {
     InvalidMeta(&'static str),
     /// Invalid drop function in vtable.
     InvalidDropFn(FnSig<'tcx>),
+    /// Invalid size in a vtable: too large.
+    InvalidSize,
+    /// Invalid alignment in a vtable: too large, or not a power of 2.
+    InvalidAlignment(String),
```

Comment:
I feel like `InvalidVtable{Size,Alignment,DropFn}` would be clearer names for these variants. What do you think?

Reply:
sure, I'll do that ! I initially did name those specifically for vtables but then `InvalidDropFn` "convinced me" not to do so ^^